packages/amplify-cli/src/lib/initialize-env.js

```diff
@@ -9,6 +9,11 @@ const { getProviderPlugins } = require('../extensions/amplify-helpers/get-provid
 
 async function initializeEnv(context, currentAmplifyMeta) {
   const currentEnv = context.exeInfo.localEnvInfo.envName;
+  const isPulling =
+    context.input.command === 'pull' ||
+    context.input.command === 'push' ||
```

Comment:
variable name suggests pulling, but a valid value is "push", what am I missing?

Reply:
because "push" initially does a "pull", so when the execution reaches this part of the code from a push command, it's also pulling.